### ISSUE TITLE
current master misses a dev-dep (eslint-loader)

### ISSUE THREAD

**Issue by birdspider** [IHCantabria/Leaflet.CanvasLayer.Field#5]
missing dep:

```
✔ 13:15 ./Leaflet.CanvasLayer.Field [master {origin/master}|✔] $ webpack -p 
Hash: 9cd8aefc2e75112ed115
Version: webpack 2.5.1
Time: 35ms

ERROR in Entry module not found: Error: 
  Can't resolve 'eslint-loader' in './Leaflet.CanvasLayer.Field/src'

✘-2 13:15 ./Leaflet.CanvasLayer.Field [master {origin/master}|✔] $ 
  npm install eslint-loader --save-dev
```

**Comment by VictorVelarde** [comment]
Thanks @birdspider 
A mistake while migrating from webpack 1 to 2

**Comment by VictorVelarde** [comment]
Fixed with latest commit & v1.0.0
